node/src/components/linear_chain_fast_sync.rs

```diff
@@ -53,6 +58,15 @@ impl<I: Clone + PartialEq + 'static> LinearChainFastSync<I> {
         })
     }
 
+    /// Initialize `LinearChainSync` component from preloaded `State`.
+    pub fn from_state(
+        _registry: &Registry,
+        _chainspec: &Chainspec,
+        _state: State,
+    ) -> Result<Self, prometheus::Error> {
+        panic!("not implemented yet for fast sync")
```

Comment:
Rather than a static method, we can roll this up in to the `new` constructor for `LinearChainSync` and then we don't need this sentinel.

Reply:
Good idea.

Reply:
Done in [`087dddd` (#904)](https://github.com/CasperLabs/casper-node/pull/904/commits/087dddd78b6b23012eca2d7f765adf874623c248).